<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Tests\Unit\UserTest_C042_check_N_numerical" file="C:\laravel_paiza\tests\Unit\UserTest_C042_check_N_numerical.php" tests="1" assertions="0" errors="1" warnings="0" failures="0" skipped="0" time="0.011429">
    <testcase name="test_C042_check_N_numerical" class="Tests\Unit\UserTest_C042_check_N_numerical" classname="Tests.Unit.UserTest_C042_check_N_numerical" file="C:\laravel_paiza\tests\Unit\UserTest_C042_check_N_numerical.php" line="16" assertions="0" time="0.011429">
      <error type="PHPUnit\Framework\Exception">Tests\Unit\UserTest_C042_check_N_numerical::test_C042_check_N_numerical
PHPUnit\Framework\Exception: Class "App\Http\Controllers\Exception" does not exist</error>
    </testcase>
  </testsuite>
</testsuites>
